blueflood-core/src/test/java/com/rackspacecloud/blueflood/io/serializers/IMetricSerializerTest.java

```diff
@@ -73,7 +73,6 @@ public void testSetSerialization() throws IOException {
 
         BluefloodSetRollup setDeserialized = mapper.readValue(setValue, BluefloodSetRollup.class);
         String setSerialized = mapper.writeValueAsString(setDeserialized);
-        Assert.assertEquals(setValue, setSerialized);
```

Comment:
I think the way a set is serialized in Java 8, is a little bit different than in JAva 7. I am ok with removing this assert since we are comparing the 2 objects ultimately. However, this type of test is all over in Blueflood and we may run into similar issues later too.